app/stac_api/models.py

```diff
@@ -182,30 +182,21 @@ def update_geoadmin_variants(self, asset_geoadmin_variant, asset_proj_epsg, asse
         '''
         try:
 
-            logger.debug(
-                "updating geoadmin:variants, self.summaries['eo:gsd']=%s, asset_eo_gsd=%s",
-                self.summaries["geoadmin:variant"],
-                asset_eo_gsd
-            )
+            # logger.debug(
+            #     "updating geoadmin:variants, self.summaries['eo:gsd']=%s, asset_eo_gsd=%s",
+            #     self.summaries["geoadmin:variant"],
+            #     asset_eo_gsd
+            # )
 
-            if self.summaries["geoadmin:variant"] is None:
-                self.summaries["geoadmin:variant"] = [asset_geoadmin_variant]
-                self.save()
-            elif asset_geoadmin_variant not in self.summaries["geoadmin:variant"]:
+            if asset_geoadmin_variant not in self.summaries["geoadmin:variant"]:
                 self.summaries["geoadmin:variant"].append(asset_geoadmin_variant)
                 self.save()
 
-            if self.summaries["proj:epsg"] is None:
-                self.summaries["proj:epsg"] = [asset_proj_epsg]
-                self.save()
-            elif asset_proj_epsg not in self.summaries["proj:epsg"]:
+            if asset_proj_epsg not in self.summaries["proj:epsg"]:
                 self.summaries["proj:epsg"].append(asset_proj_epsg)
                 self.save()
 
-            if self.summaries["eo:gsd"] is None:
-                self.summaries["eo:gsd"] = [asset_eo_gsd]
-                self.save()
-            elif not float_in(asset_eo_gsd, self.summaries["eo:gsd"]):
+            if not float_in(asset_eo_gsd, self.summaries["eo:gsd"]):
                 self.summaries["eo:gsd"].append(asset_eo_gsd)
                 self.save()
```

Comment:
Nice idea setting the defaults for the summaries to `[]` instead of `None`. Makes this if-block much more elegant :smile: